### PR TITLE
Add no-get-in-getters-name tenet

### PR DESCRIPTION
**Pull request by leilaes**

### CODE REVIEW

File: tenets/codelingo/effective-go/no-get-in-getters-name/codelingo.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    body: |
+      function(name) {
+        return name.substring(0,3) === "Get"
+      }
+
+  - name: fixName
+    type: resolver
+    body: |
+      function(name) {
+        return name.charAt(0).toUpperCase() + name.substring(1)
```
Comment (Daanikus) [inline tenets/codelingo/effective-go/no-get-in-getters-name/codelingo.yaml]:
```suggestion
        name = name.substring(3)
        return name.charAt(0).toUpperCase() + name.substring(1)
```

Comment (Daanikus) [reply tenets/codelingo/effective-go/no-get-in-getters-name/codelingo.yaml]: Does that look right? We wanna strip the "Get" I assume. And missing rewrite decorator. Have you tested the rewrite or are you postponing it?

Comment (Daanikus) [reply tenets/codelingo/effective-go/no-get-in-getters-name/codelingo.yaml]: Callgraph might work for this

Comment (leilaes) [reply tenets/codelingo/effective-go/no-get-in-getters-name/codelingo.yaml]: `name` here is the "field" of the struct, not Getter method name and wouldn't have "Get" in it. I have not tried rewrite yet and have postponed it for now. Is it priority?

Comment (Daanikus) [reply tenets/codelingo/effective-go/no-get-in-getters-name/codelingo.yaml]: No, not a priority. You got a LGTM so you can leave it

**Pull request merged**